lib/tizen_project.dart

```diff
@@ -21,15 +26,39 @@ class TizenProject extends FlutterProjectPlatform {
 
   final FlutterProject parent;
 
+  /// See: [FlutterManifest] in `flutter_manifest.xml`
+  late final Map<String, Object?> _parentPubspec = () {
```

Comment:
We can't use `parent.manifest` directly and should parse the file again because `FlutterManifest._descriptor` is private.